docs/figure_file_format.rst

```diff
@@ -129,6 +129,15 @@ Optional settings for each panel::
             "size": "12",
             "position": "topright",
             "color": "FFFFFF"
+        },
+        {
+            // for 'time', 'x', 'y', 'width' and 'height', decimal precision
+            // can be specified in 3rd position (here 2)
+            // 'time' can be relative to a frame, specified after a '-' sign (here 3)
+            "text": "Time (s): [time.secs.2-3]",
```

Comment:
Just as a heads up, my inclination is that this format won't be readily understandable by users. Sorry if I missed it, but was there any discussion of the cost/benefit trade-off on having a larger but more self-documenting format

Reply:
Brainstorming:
 - `time[3].{unit:.2f}` (more python-y)
 - `[time.unit;offset=3;precision=2]` (more readable)

Reply:
`[time.unit;offset=3;precision=2]`   I like that one. It's readable, won't be much effort to implement and we don't need to worry about the order of parameter